recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "colorcet" %}
-{% set version = "2.0.6" %}
-{% set sha256 = "efa44b6f4078261e62d0039c76aba17ac8d3ebaf0bc2291a111aee3905313433" %}
+{% set version = "3.0.0" %}
+{% set sha256 = "21c522346a7aa81a603729f2996c22ac3f7822f4c8c303c59761e27d2dfcf3db" %}
 
 package:
   name: {{ name|lower }}
```

Comment:
Could you please remove `noarch: python` and rebuild it as arch-specific?
Then also update python in `host` and `run` as a plain `python` without pinning
It would be nice to add `pip` in `test/requires` and `pip check` in `test/commands`